dev/com.ibm.ws.session.cache_fat/fat/src/com/ibm/ws/session/cache/fat/SessionCacheTimeoutTest.java

```diff
@@ -64,6 +64,17 @@ public void testInvalidationTimeout() throws Exception {
         appOnelistener.sessionGet("testInvalidationTimeout-foo", null, session);
     }
 
+    /**
+     * Test that a session can still be used if it was valid when a servlet call began, even after timeout.
+     * This mimics SessionDB behavior.
+     */
+    @Test
+    public void testServletTimeout() throws Exception {
+        List<String> session = new ArrayList<>();
+        appOnelistener.sessionPut("testInvalidationTimeout-foo2", "bar", session, true);
+        appOnelistener.sessionGetTimeout("testInvalidationTimeout-foo2", "bar", session);
```

Comment:
Instead of making a new method on the app class that only gets used once, lets replace it with:
```java
invokeServlet("sessionGetTimeout&key=testInvalidationTimeout-foo2&expectedValue= bar", session);
```
(and just assume that the type is `String` on the servlet side)
If you think we would re-use this method in more places, we can keep it though.